perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -682,6 +682,20 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 **
+	 * Get list of all richGroups with selected attributes assigned to the resource filtered by specific member.
+	 * Allowed namespaces of attributes are group, group-resource and member-group.
+	 *
+	 * Last step is filtration of attributes:
```

Comment:
We should probably rewrite this message to by more RPC API like.

You will get only attributes which you are authorized to read. You must specify names of requested attributes by their URNs in ``attrNames``. You can pass empty list to return all possible attributes.

Reply:
Javadoc and the processing of code was changed.